bin/delete-record.js

```diff
@@ -36,7 +36,14 @@ ovh.request('GET', `/domain/zone/${dom.domain}.${dom.tld}/record`, {
       console.error(deleteErr);
       process.exit(2);
     }
+  });
 
-    process.exit(0);
+  ovh.request('POST', `/domain/zone/${dom.domain}.${dom.tld}/refresh`, (refreshErr) => {
+    if (refreshErr) {
+      console.error(refreshErr);
+      process.exit(3);
+    }
   });
+
+  process.exit(0);
```

Comment:
Well, now this is even worse 😅
Basically what will happen is:

1. DELETE request is scheduled
2. POST request is scheduled
3. Process exited (0)
4. DELETE & POST request are executed at the same time.

Of course 4 never happens because the process is already exited. The way it was done before it was perfect. Before, the flow was:

1. DELETE is scheduled
2. DELETE is executed
3. Callback of DELETE is executed
4. POST is scheduled
5. Process is exited (0)
6. POST is executed.

The problem here was simply that the process exited before the refresh call could be executed. I suggest to go back to the previous commits, simply deleting the `process.exit(0);` line that is superfluous and actually bad.

Reply:
Okay, to be clear: you wrote the `process.exit(0);` not me. That actually confused me a bit.
What confuses me a little bit more, the code actually works fine.

Reply:
I know, I wrote it, but when I did there was nothing else to do… while now you have also refresh the zone — which btw is a good idea because it is a necessary step to update the zone.

In my tests it actually doesn't delete the record… you can test by running dig against the DNS record or using the API console here: https://api.ovh.com/console/

Reply:
Okay, I have to excuse. You are right, the record will not be deleted. It is possible that I ran the test with older code... oops.